.idea/runConfigurations.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="RunConfigurationProducerService">
+    <option name="ignoredProducers">
+      <set>
+        <option value="org.jetbrains.plugins.gradle.execution.test.runner.AllInPackageGradleConfigurationProducer" />
+        <option value="org.jetbrains.plugins.gradle.execution.test.runner.TestClassGradleConfigurationProducer" />
+        <option value="org.jetbrains.plugins.gradle.execution.test.runner.TestMethodGradleConfigurationProducer" />
+      </set>
+    </option>
+  </component>
+</project>
```

Comment:
Prefer not adding this since it'll affect everyone.

Reply:
I accidentally modified this file in one of the commits, so I reverted this file back to its original state in my last commit. I used `git checkout commitId -- .idea/runConfigurations.xml` to do this is it the right way to handle these kinds of situations?

Reply:
Generally, I recommend not using commands like ``git commit -a`` since then you can carefully control what you stage for commit. My workflow is usually:

```
git add app/*
git add domain/*
...
git status
git commit
```

(And then I fill in the error message using a command-line text editor). This helps me avoid adding unrelated files. Note that the 'git status' specifies exactly which files will be committed.

Reply:
thanks.